docs/attributes-registry/db.md

```diff
@@ -170,23 +170,23 @@ This group defines attributes for Azure Cosmos DB.
-| `Batch`             | batch              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Create`            | create             | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Delete`            | delete             | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Execute`           | execute            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `ExecuteJavaScript` | execute_javascript | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Head`              | head               | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `HeadFeed`          | head_feed          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Invalid`           | invalid            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Patch`             | patch              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Query`             | query              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `QueryPlan`         | query_plan         | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Read`              | read               | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `ReadFeed`          | read_feed          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Replace`           | replace            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `Upsert`            | upsert             | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Value                | Description        | Stability                                                        |
+| -------------------- | ------------------ | ---------------------------------------------------------------- |
+| `batch`              | batch              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `create`             | create             | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `delete`             | delete             | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `execute`            | execute            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `execute_javascript` | execute_javascript | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `head`               | head               | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `head_feed`          | head_feed          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `invalid`            | invalid            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `Patch`              | patch              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
nit: is there a reason this is upper case and the others are lower case?

Reply:
great catch, thanks!